blog/content/post/2023-09-12-simplifying-firmware-update.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Simplifying Firmware Updates in Dasharo'
```

Comment:
I believe I wrote this only in Jira, but please change the title to something like: 'Optimizing Firmware Updates: Dasharo's Firmware Update Mode for NovaCustom Laptops. WDYT @macpijan

Reply:
Fixed in 4c668f3b2b9c72cecbb77c52366aacbc7783a1ca. Let's continue the review in the other (open) PR